src/Components/Dropdowns/DropdownItem.tsx

```diff
@@ -21,6 +21,8 @@ export interface DropdownItemProps extends StandardFunctionProps {
   wrapText?: boolean
   /** Title attribute */
   title?: string
+  /** Prevents any interaction with this element, including the onClick function */
+  disabled?: boolean
```

Comment:
Should we use `status` & `ComponentStatus.Disabled` here like we do in our other components? This would also allow us to disable it during some loading state and we could use it to toggle to active state styles.